src/23-2017-q1-update/index.md

```diff
@@ -0,0 +1,62 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 2017-q1-update
+
+breadcrumbs:
+  - {name: "2017-q1-update", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2017-01-25
+
+# this is the Title
+title: IPFS Project Management Update for Q1 2017
+
+# this is the name of the main author(s)
+author: Richard Littauer
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+# IPFS going into Q1 in 2017
+
+This year, IPFS is changing the way we do our sprints. This post covers some of the basic aspects of these changes. A longer post will be coming in February.
+
+In short, we've switched from weekly set-your-own-goal sprints focused on particular areas - js-ipfs, go-ipfs, Apps on IPFS, and so on - and we're moving, as a project, to more defined two week sessions where we work together on scrum-like projects.
+
+We talked about this in the All Hands call on Monday the 16th. You can see the video for that, here:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Cvz_PW1GfIY" frameborder="0" allowfullscreen></iframe>
+
+### Why the change?
+
+<!-- TODO: Should also cover lessons learned about kanban (loosely coordinated vs. not coordinated) -->
+
+Last year, our core contributor base grew immensely. There are now dozens of contributors working at least part-time on IPFS and IPFS related projects (e.g. cluster, multiformats, IPLD, Orbit). With the amount of contributors, the amount of projects we've been actively working on has grown, almost exponentially.
+
+As you may know, Protocol Labs has had a quarterly meeting for some of the core contributors to discuss project roadmaps for that quarter. Last year, during our fourth quarter, we set very ambitious goals for dozens of little projects. At our Lab Week in Berlin this January, we looked back at our projects - and found that, as a team, we hadn't been able to meet all of those goals.
+
+We think that a large part of why we didn't achieve everything was that it was difficult to see what everyone else was working on, and because we were constantly context-switching between dozens of little projects. This lead to thrashing - like a processor, there were too many active threads going on at any one point, which led to limited bandwidth for getting real work done.
+
+So, we're pivoting; instead of having everyone come up with their own goals and ideas, and syncing with the team casually about what goals should go where, we're going to be much more explicit about what goals we can do as a project, and who can work together.
+
+### What will this look like?
+
```

Comment:
This is an experiment. We will follow this arrangement at the end of Q1 to consider whether it's working and where we an improve. For this first run of the experiment, we've scheduled three types of things on our roadmap: [sprints](https://github.com/ipfs/pm/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3Asprint), [sprintinos](https://github.com/ipfs/pm/issues?q=is%3Aopen+is%3Aissue+label%3Asprintino) and [events](https://github.com/ipfs/pm/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3Aevent%20)

For Q1, our definition of a [sprint](https://github.com/ipfs/pm/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3Asprint) is a team effort that
* lasts exactly two weeks
* involves 3 or more team members
* is the main focus of that team's attention for the duration of the 2 weeks
* has a Captain/PM assigned to lead it

By contrast, a [sprintino](https://github.com/ipfs/pm/issues?q=is%3Aopen+is%3Aissue+label%3Asprintino) lasts one week or less and involves 1 or 2 team members who are making it the main focus of their attention.

The [events](https://github.com/ipfs/pm/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3Aevent%20) are any events that IPFS team members will be attending, which impacts the work we do that week and also gives community members an opportunity to know about opportunities to meet and interact in-person.